hooks/push

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -ex
+
+LASTEST_MAJOR="3"
```

Comment:
This could avoid being hardcoded (since we are guaranteed to forget this exists) by doing something like:

```bash
git tag --list | tr --delete 'v' | sort --general-numeric-sort | tail --lines 1 | \
   grep --extended-regex --only-matching '^[0-9]+'
```

Reply:
Good point!